modules/config-idp.adoc

```diff
@@ -16,53 +19,72 @@ Configuring GitHub authentication allows users to log in to {product-title} with
++
+The following is an example of a homepage URL for a GitHub identity provider:
 +
 ----
-https://oauth-openshift.apps.<cluster_name>.<cluster_domain>/oauth2callback/<idp_provider_name>
+https://oauth-openshift.apps.example-openshift-cluster.com
 ----
-+
-For example:
+
+* For the authorization callback URL in your GitHub OAuth app configuration, specify the full *OAuth callback URL* that is automatically generated in the *Add a GitHub identity provider* page on {OCM}. The full URL has the following syntax:
 +
 ----
-https://oauth-openshift.apps.example-openshift-cluster.com/oauth2callback/github/
+https://oauth-openshift.apps.<cluster_name>.<cluster_domain>/oauth2callback/<idp_provider_name>
 ----
 
-. link:https://docs.github.com/en/developers/apps/creating-an-oauth-app[Register an application on GitHub].
+. Return to the *Edit identity provider: GitHub* dialog in {OCM} and select *Claim* from the *Mapping method* drop-down menu.
 
-. Return to {product-title} and select a mapping method from the drop-down menu. *Claim* is recommended in most cases.
+. Enter the *Client ID* and *Client secret* for your GitHub OAuth application. The GitHub page for your OAuth app provides the ID and secret.
 
-. Enter the *Client ID* and *Client secret* provided by GitHub.
-
-. Enter a *hostname*. A hostname must be entered when using a hosted instance of GitHub Enterprise.
+. Optional: Enter a *hostname*.
++
+[NOTE]
+====
+A hostname must be entered when using a hosted instance of GitHub Enterprise.
+====
 
-. Optional: You can use a certificate authority (CA) file to validate server certificates for the configured GitHub Enterprise URL. Click *Browse* to locate and attach a *CA file* to the identity provider.
+. Optional: You can specify a certificate authority (CA) file to validate server certificates for a configured GitHub Enterprise URL. Click *Browse* to locate and attach a *CA file* to the identity provider.
 
-. Select *Use organizations* or *Use teams* to restrict access to a particular GitHub organization or a GitHub team.
+. Select *Use organizations* or *Use teams* to restrict access to a GitHub organization or a GitHub team within an organization.
 
-. Enter the name of the organization or team you would like to restrict access to. Click *Add more* to specify multiple organizations or teams that users can be a member of.
+. Enter the name of the organization or team you would like to restrict access to. Click *Add more* to specify multiple organizations or teams.
++
+[NOTE]
+====
+Specified organizations must own an OAuth app that was registered by using the preceding steps. If you specify a team, it must exist within an organization that owns an OAuth app that was registered by using the preceding steps.
+====
 
-. Click *Confirm*.
+. Click *Save* to apply the identity provider configuration.
```

Comment:
It's "Add" button rather than `Save`

Reply:
Thank you for highlighting this. Resolved.